frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -302,16 +303,28 @@ const composeAndPushNotificationMessages = (
             getSelectedRole()?.role_name
         );
     }
-    if (errorIdList.length > 0) {
+
+    if(errorMsg.msg) {
         setGrantAccessNotificationMsg(
-            errorIdList,
+            errorMsg.errorForestClientIdList,
+            username,
+            Severity.error,
+            getSelectedRole()?.role_name,
+            `An error has occured. ${errorMsg.msg}`
+        );
+    }
+
+    if (errorMsg.errorForestClientIdList.length > 0) {
+        setGrantAccessNotificationMsg(
+            errorMsg.errorForestClientIdList,
             username,
             Severity.error,
             getSelectedRole()?.role_name
         );
     }
```

Comment:
So if the `composeAndPushNotificationMessages()` method receives this `errorMsg` 
```
{
msg: "Yo, why don't you give me permissions?",
errorForestClientIdList: [
   "00000000",
   "99999999"
   ]
}
```
How does this "**composeAndPushNotificationMessages**()" method pops out the notifications?

Reply:
Please be careful validate the code line by line.
This is what I can produce and not the message you expect to see.
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/50c056aa-d8f1-4e1d-9b74-b8ce5d865bb6)

Reply:
How did you produce that?

Reply:
@J0taFerreira why not look at my original comment and this comment thread to understand how I arrive this instead? I already explain this at the beginning if you look carefully.
I used exact testing object from what I asked you before on how it would produce the message when the object is passed to your function.
We can talk after scrum if you still is not sure. @MCatherine1994